Client/Frontend/Browser/HomePanel/FavoritesViewController.swift

```diff
@@ -686,6 +720,6 @@ extension FavoritesViewController: FavoriteCellDelegate {
 
 extension FavoritesViewController: PreferencesObserver {
     func preferencesDidChange(for key: String) {
-        self.resetBackground()
+        self.resetBackgroundImage()
```

Comment:
This changes for every preference in the app right?

Reply:
No only:
```
Preferences.NewTabPage.backgroundImages.observe(from: self)
Preferences.NewTabPage.backgroundSponsoredImages.observe(from: self)
```
https://github.com/brave/brave-ios/pull/2203/files/9887873dbed1788a0ffc7c3f5f3d159d54d9dd73#diff-9dddcde1fc3411fdf044fc8d4e5a79d0R241